tests/common/dualtor/dual_tor_mock.py

```diff
@@ -340,12 +340,21 @@ def apply_peer_switch_table_to_dut(cleanup_mocked_configs, rand_selected_dut, mo
     peer_switch_key = 'PEER_SWITCH|{}'.format(peer_switch_hostname)
     device_meta_key = 'DEVICE_METADATA|localhost'
 
+    cmd = 'redis-cli -n 4 HSET "{}" "{}" "{}"'.format(device_meta_key, 'subtype', 'DualToR')
+    dut.shell(cmd=cmd)
+    if dut.get_asic_name() in ['th2', 'td3']:
+        # Restart swss on TH2 or TD3 platform to trigger syncd restart to regenerate config.bcm
+        # We actually need to restart syncd only, but restarting syncd will also trigger swss
+        # being restarted, and it costs more time than restarting swss
+        logger.info("Restarting swss service to regenerate config.bcm")
+        dut.shell('systemctl restart swss')
```

Comment:
Not sure if swss restart triggers regenerating config.bcm. Is this confirmed?

Reply:
Yes, it's confirmed. Restarting swss will trigger syncd being restarted, and the `start.sh` in syncd docker will regenerate the config.bcm.